index.html

```diff
@@ -37,16 +37,24 @@ <h1 id="splash-title">ROCKET MAN</h1>
     <main>
         <div class="game-wrapper">
             <div class="col" id="pos-1">
-                <div id="city-1" class="city city1" data-city="1"></div>
+                <div id="city-1" class="city city1" data-city="1">
+                    <img class="explosion">
```

Comment:
this is invalid HTML. images are not allowed to have no src

Reply:
fixed in eb102a8